codeowners/src/codeowners.rs

```diff
@@ -3,10 +3,12 @@ use std::{
     path::{Path, PathBuf},
 };
 
-use codeowners::{FromReader, GitHubOwners, GitLabOwners};
+use crate::github::GitHubOwners;
+use crate::gitlab::GitLabOwners;
+use crate::traits::FromReader;
 use serde::{Deserialize, Serialize};
 
-use crate::constants::CODEOWNERS_LOCATIONS;
+use constants::CODEOWNERS_LOCATIONS;
```

Comment:
nit: Can we group these imports together so they'll be formatted into an alphabetical list?